src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,55 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+type Tab = {
+  id: string;
+  title: string;
+  content: string;
+};
+
+type TabsProps = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }: TabsProps) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const getTabContent = () => {
+    return tabs.find(tab => tab.id === selectedTabId)?.content;
+  };
```

Comment:
This function is redundant. You have already filtered out the selectedTab above this code in line 16.